_posts/2023-07-11-missing-side-effects-on-selector-access-bug.md

```diff
@@ -0,0 +1,53 @@
+---
+layout: post
+published: true
+title: 'Bug when accessing the .selector member on complex expressions in legacy code generation'
+date: '2023-07-11'
```

Comment:
IMO the title would sound better like this:

```suggestion
title: 'Bug in legacy code generation when accessing the .selector member on expressions with side-effects'
date: '2023-07-19'
```

Also, the likely date for the release is the next Wednesday. You can change that in the file name as well.